### DISCUSSION TITLE
Using payload with or within Remix

### DISCUSSION THREAD

**Discussion by jopesh** [payloadcms/payload#780] [Q&A]
Hey there!

Has anyone successfully combined Remix(.run) and Payload in a single project yet? I've been fiddling around for a bit, but can't get it working. Maybe someone was successful already?

**Comment by DanRibbens** [comment]
We haven't had a chance to try this ourselves. Would love to see this done in an example repo if somebody has!

**Comment by GeorgeyB** [comment]
@jopesh I'm working on a project that uses a Payload-Remix combo right now. What are you having trouble with specifically?

**Reply by jopesh** [reply]
Well I'm not quite experienced with express and thus can't get the plumbing right to get both apps running at the same time. By initialising payload in the remix express server, the 'admin' route shows just a blank page and seems to blow up. What is there to consider?

**Comment by SkywaveTM** [comment]
I'm also working on Remix+Payload. As I remember, I chose `Express Server` during remix project setup with `npx create-remix`. And then I just added `payload.init({...});` statement on `server.js`, just before `app.all(~)` statement which is automatically generated by Remix.

Anyway, using payload's local API on remix's loader functions is quite cool.
